bin/live/pycbc_live_combine_single_fits

```diff
@@ -37,6 +40,14 @@ args=parser.parse_args()
 
 pycbc.init_logging(args.verbose)
 
+# Assert some sensible limits on the arguments
+
+if args.conservative_percentile < 50 or \
+    args.conservative_percentile > 99:
+    raise parser.error("--conservative-percentile must be between 50 and 99, "
+                       "otherwise it is either not a percentile, or not "
+                       "conservative.")
```

Comment:
```suggestion
    parser.error("--conservative-percentile must be between 50 and 99, "
                 "otherwise it is either not a percentile, or not "
                 "conservative.")
```